vnext/ReactUWP/Modules/I18nModule.h

```diff
@@ -10,19 +10,42 @@
 namespace react {
 namespace uwp {
 
+class I18nHelper;
+
 class I18nModule final : public react::windows::II18nModule {
  public:
   using I18nInfo = std::pair<std::string, bool>;
   static I18nInfo GetI18nInfo(); // Must be called from a UI thread
 
   // II18nModule
-  I18nModule(I18nInfo &&i18nInfo);
+  I18nModule();
 
   std::string getLocaleIdentifier() override;
   bool getIsRTL() override;
+  void setAllowRTL(bool allowRTL) override;
+  void setForceRTL(bool forceRTL) override;
 
  private:
-  I18nInfo m_i18nInfo;
+  I18nHelper &m_helper;
 };
+
+class I18nHelper {
+ public:
+  static I18nHelper &Instance();
+
+  I18nHelper();
+
+  void setInfo(I18nModule::I18nInfo &&i18nInfo);
+  std::string getLocaleIdentifier();
+  bool getIsRTL();
+  void setAllowRTL(bool allowRTL);
+  void setForceRTL(bool forceRTL);
+
+ private:
+  I18nModule::I18nInfo m_i18nInfo;
+  bool m_allowRTL = true;
```

Comment:
> = true [](start = 17, length = 7)

The latest 'fashion' is to use the universal initializers: 
bool m_allowRTL{true};

E.g. the std::atomic fails to initialize otherwise in Clang.